### ISSUE TITLE
Add 'Earn Bitcoin' section

### ISSUE THREAD

**Issue by beyourseff** [bitcoin-dot-org/bitcoin.org#2524]
**Introduction**: Not everyone who's interested in Bitcoin, is able to buy Bitcoin. I propose the Bitcoin.org website gets updated to include information on how to go about earning Bitcoin.

**Explanation**: As mentioned above, not everyone has the financial resources to purchase Bitcoin. This can vary from people living in 2nd/3rd world countries to those who live paycheck to paycheck in 1st world countries. Bitcoin should be obtainable by anyone, this also includes the financially less fortunate.

To include the above groups in the financial revolution of Bitcoin, there should be a way _(ideally multiple ways)_ for them to earn Bitcoin. An existing example of this is [CoinMall](https://www.coinmall.com/), a globally accessible marketplace on which anyone can sell digital products for cryptocurrencies. Other examples would include similar, globally accessible, earning platforms such as (Bitcoin-powered) Amazon MTurk, Fiverr and the like.

The added benefit of educating and assisting said individuals in earning Bitcoin, is that it will help them in earning a living, providing for their families. The subsequent effect is that people from within their communities will start to get interested in Bitcoin, and incentivizes them to also partake in the Bitcoin ecosystem.

**Steps to take**:
- Create a 'earn' page with information and resources
- Add a link to the 'earn' page to the 'Participate' dropdown menu
- Add a 'Earn Bitcoin' button on the home page, possibly next to the 'Buy Bitcoin'

**Pros:**
- It will help newcomers with little/no money to partake in the ecosystem and possibly earn a living
- It will educate them, and subsequently they will educate their communities who will grow interested in Bitcoin as well (the Bitcoin virus spreads)
- The resources they offer may be beneficial to current hodlers, who'll be incentivized to spend a little BTC.
- ~~World domination. Pinky & The Brain plan has succeeded.~~

**Cons:**
- None

**Comment by wbnns** [comment]
@beyourseff Hey, thanks for mentioning this. Yes, would be great if we could find more ways to show people how they can earn bitcoin, so they can work from anywhere in the world, whenever they want.

**Comment by Cobra-Bitcoin** [comment]
I like the idea of an earn Bitcoin page, sounds like it could help many people. I'm sure lots of people search for "earn Bitcoin" on search engines too, and we could also recommend helping translate Bitcoin.org as a good way to earn a little bit. I don't think we should have a prominent button on the homepage though, there's too many buttons on there as it is.

**Comment by beyourseff** [comment]
@Cobra-Bitcoin @wbnns Many thanks for the feedback guys, glad you agree that it's an important aspect.

To make the proposal a bit more concrete, I'd like to amend the following things:
- Not add the 'Earn Bitcoin' button (there are indeed too many buttons as Cobra has mentioned)
- Add an earn link to the footer (under the 'Participate' header)

In addition to the above, the earn page itself should have the following:
- Intro: short description how users can earn Bitcoin globally with no restrictions
- Ways to earn: display a few ways to earn Bitcoin. I was thinking we should have a link to the available Bitcoin.org bounties at the top, because it both requires no resources and improves the site simultaneously. The second way would be CoinMall.com, and then there can be other ways added as they get proposed. A possible third addition could be earn.com

There should be a few conditions as to which earning platforms should be added in the future. Here's what I was thinking of:
- Is the platform globally accessible?
- Does the platform prominently focus on earning Bitcoin (and not **aggressively** advocate for using other cryptocurrencies by making up false narratives against Bitcoin)
- Does the platform respect the privacy of the users? Think of no KYC, id verification and whatnot required before being able to use the platform

Do let me know what you think of the above, and what's required for the next steps. I can be of help with regards to intro text and short descriptions below each earning method.

**Comment by beyourseff** [comment]
@Cobra-Bitcoin Ah, my apologies, I thought I included that in the main post.

Full disclosure: I'm the founder of CoinMall. We're completely bootstrapped and currently have no outside investors.

I've been thinking of other platforms which could be added, and I think OpenBazaar might fit the bill.

**Comment by wbnns** [comment]
@beyourseff Hello, yes - having something on the site that helps people find ways they can earn bitcoin (as opposed to just buying it or using an exchange) is on the radar. I'll reference this issue or comment here when there are updates.

**Comment by beyourseff** [comment]
Hey there @wbnns @Cobra-Bitcoin - I can see that there's been a bounty assigned, has anyone started working on this yet?

**Comment by wbnns** [comment]
@beyourseff 

Hello, no, not yet.

**Comment by beyourseff** [comment]
Hey @wbnns @Cobra-Bitcoin I've made a concept page which is attached. Any feedback of what should be added/amended?
![earn-page](https://user-images.githubusercontent.com/32877881/64059154-62921680-cbae-11e9-815b-cbdd43978772.png)

**Comment by wbnns** [comment]
@beyourseff 

Thanks for working on this and sharing the perspective / idea. Not sure the site should be the first thing people are recommended to peruse as an option to earn bitcoin; there are a limited number of bounties. For the second and third sections, there may be other options beyond CoinMail and Jobs4Bitcoin that we should consider connecting people with.

For example, just on the topic of jobs, there are a growing number of places people can go to browse opportunities to potentially earn bitcoin - here's a rough, unreviewed list in no particular order:

 + https://bitcointalk.org/index.php?board=52.0 
 + https://www.cryptogrind.com/
 + https://bitgigs.com/
 + https://www.reddit.com/r/Jobs4Bitcoins/ (you mentioned this one) 
 + https://www.reddit.com/r/BitcoinJobs/
 + https://angel.co/bitcoin/jobs
 + https://cryptojobslist.com/
 + https://microlancer.io/
 + https://dream.ac/freelance
 + https://workingforbitcoins.com/
 + https://freelanceforcoins.com/
 +  https://www.xbtfreelancer.com/ 

Also, freelancers can use services to convert their fiat-based receivables to bitcoin, so that they can earn bitcoin through their existing work:

+ https://www.bitwage.com/
+ https://freelance.kable.co/

[BTCPay](https://btcpayserver.org/) might be something else to mention as well, which would enable people to earn bitcoin for a number of [use cases](https://docs.btcpayserver.org/btcpay-basics/usecase).

To summarize, not sure the page would be realizing much of its potential to help people earn bitcoin, if we weren't connecting people with as many hopefully decent opportunities to review as possible. Not sure this layout is ideal for it.

**Comment by beyourseff** [comment]
@wbnns - No worries, I figured I'd hit two birds with one stone by getting CoinMall added (as you know, I'm its founder) and get the available bounty. Based on your feedback;

What if I subdivide the page in three distinct categories? Something like:
1. Freelancing / jobs
2. Selling products (both virtual and otherwise)
3. Microtasks / Faucets (?)

Each category would have a number of services listed with a short description for every service (up to 3? 5?). I was also thinking of BTCPay, but it's a payment processor and requires quite a bit of effort to set up for potentially relevant "earn" use-cases.

I've got a few free hours this week in which I can work on this, so any thoughts/suggestions for next steps to take would be welcome :)

**Comment by wbnns** [comment]
@beyourseff 

Thanks for the additional thoughts. Let's wait a bit to see if there are any other people that would like to weigh-in with their feedback.

**Comment by jdabs** [comment]
I do like this idea, especially as the monetary value of one Bitcoin can feel daunting to a lot of people, especially in areas with limited incomes and poor banking systems. Every little bit can be very helpful.

As to the placement, I do not see anything wrong with adding it on the particpate main menu item. The menu in the footer is the same as the one in the top nav, so it doesn't feel natural to break that flow.

As to subsections, I lean more towards loosely defined classifications of the ways to earn. To be clear, I believe that it should include job sites that are already popular but also allow payments in bitcoin. I know fivvr did before but I think that option was removed, last I checked. Any other similar services in other countries should be included.

**Comment by DrBeta** [comment]
Is this issue still open for completion

**Comment by CcliqueE** [comment]
In order to go with the concept of a beginner-friendly page for optimal search results, I think it would be beneficial to put each link card from the easiest (on the far left) to the harder or less beginner-friendly products or services on the right, for more clear and subconscious viewing experience. This way the minority of users who see something difficult and then click off may stay for longer than usual.

As well I believe there should be the addition of a Defi option at the forefront of the 'earn' page, being that anyone interested in Bitcoin itself, should for sure be exposed to the basics of earning by interest. Though the technicalities of these Defi services are for sure not beginner-friendly, the services themselves are extremely straightforward, being that all you do is buy or deposit your coins into an account that earns.

Though the services that offer APR by lending to other users may be against Bitcoins' entire theory in general, this does not aid that philosophy behind Bitcoin, with it tending to be more centralized than not. But also this may differ from service to service so I may be wrong about that.

**Comment by saiy2k** [comment]
Wow. This is something that will be very helpful to many. Actually I myself was googling ways of earning Bitcoin. I will come up with a set of relevant links and a potential page layout and revert this weekend.

**Comment by saiy2k** [comment]
Hi,
I spent an hour googling for relevant links and also consolidated the links provided by @wbnns at https://docs.google.com/spreadsheets/d/1NX5cLMcNtQyJf79K9_UPL5iPUVcarNUwU_1Xig5a2vs/edit?usp=sharing 

The sheet only has some basic structure set. Actual evaluation of services where people can Earn bitcoin is an exhaustive process. Like to know if the bounty is still ON? If yes, can you please update the bits/sats or USD with current market rates?

I will happy to do this, if the bounty is good enough. Thanks.

**Comment by TheTmelt** [comment]
This is awesome. The way and "the one" is proof of how money will evolve. This includes paychecks and payrolls. Goods and services will forever be digital or, as I like to say, Earned through reputation.

**Comment by Croser123** [comment]
Hello guys

**Comment by Solomon4crypto** [comment]
Someone thinking like me, 🤣🤔 so it true when you think you are the only one with the idea there's someone, either pushing it or already has it done🤣🤣🤣🤣😁😁♥️♥️♥️♥️

**Comment by Daisuke134** [comment]
I implemented this in PR #4720: https://github.com/bitcoin-dot-org/Bitcoin.org/pull/4720\n\nIncluded:\n- new Earn Bitcoin page\n- Participate menu link\n- home-page Earn Bitcoin button

**Comment by caydyan** [comment]
Submitted an implementation for this bounty: https://github.com/bitcoin-dot-org/Bitcoin.org/pull/4790

This version adds an Earn Bitcoin page, links it from the Participate navigation/footer, and focuses on practical ways to earn bitcoin through work, selling, payments, and contribution paths while avoiding custodial yield or faucet-style recommendations.

If accepted for the bounty, BTC payout address: `bc1qev5ant33v5y89qqjvcf4mh9hlax5svqf5xd7gc`
